exchanges/okx/okx_websocket.go

```diff
@@ -1631,6 +1920,72 @@ func (ok *Okx) WsAmendMultipleOrders(args []AmendOrderRequestParams) ([]OrderDat
 	}
 }
 
+// WsMassCancelOrders cancel all the MMP pending orders of an instrument family.
+// Only applicable to Option in Portfolio Margin mode, and MMP privilege is required.
+func (ok *Okx) WsMassCancelOrders(ctx context.Context, args []CancelMassReqParam) (bool, error) {
```

Comment:
This doesn't appear to take in multiple CancelMassReqs
https://www.okx.com/docs-v5/en/#order-book-trading-trade-post-mass-cancel-order
Have you had success in your testing?
Also missing `lockInterval`